src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/isl/IslHandler.java

```diff
@@ -164,6 +172,12 @@ public void islDefaultRulesDelete(Endpoint source, Endpoint destination) {
                 new SpeakerRulesIslRemoveCommand(keyFactory.next(), source, destination)));
     }
 
+    @Override
+    public void unhurriedPollUpdateRequest(Endpoint endpoint, boolean enableUnhurriedPoll) {
+        WatchListCommand command = new WatchListUnhurriedPollUpdateCommand(endpoint, enableUnhurriedPoll);
+        emit(STREAM_POLL_ID, getCurrentTuple(), makePollTuple(command));
```

Comment:
Check for possible loops produced by this stream. Perhaps we must not tie it to `getCurrentTuple()`.

Reply:
This anchor has been removed in this place.